src/DGmethods/DGmodel.jl

```diff
@@ -79,9 +79,24 @@ function (dg::DGModel)(dQdt, Q, ::Nothing, t; increment=false)
                            vgeo, sgeo, t, vmapM, vmapP, elemtobndy,
                            topology.realelems))
 
-    communicate && MPIStateArrays.start_ghost_exchange!(Qvisc)
+    if communicate
+      MPIStateArrays.start_ghost_exchange!(Qvisc)
+    end
+
+    if true
+      if communicate
+        MPIStateArrays.finish_ghost_recv!(Qvisc)
+      end
+
+      update_aux_diffusive!(dg, bl, Q, t)
+
+      if communicate
+        MPIStateArrays.start_ghost_exchange!(auxstate)
+      end
+    end
```

Comment:
@jkozdon, is this what you wanted for the communication?

Reply:
I was thinking something like this

https://github.com/climate-machine/CLIMA/commit/44157d06e78d8b0e4099203450e628afa47846d5

@simonbyrne might have a better suggestion.

Reply:
Note my code won't run since all the existing `update_aux!` functions don't return `Bool`, but this way we just communicate `auxstate` as needed.

Reply:
Okay, I implemented something similar to what you did and it works